scripts/components/Dialog/PasswordContent.js

```diff
@@ -117,9 +117,10 @@ export default class PasswordContent extends React.Component {
               />
             </div>
             {this.props.hint && (
-              <span className={"h5p-field-text"}>
-                {`${this.context.l10n.hint}: ${this.props.hint}`}
-              </span>
+              <div className={"h5p-field-text"}>
+                {this.context.l10n.hint}:
+                <div dangerouslySetInnerHTML={{ __html: this.props.hint }} />
```

Comment:
Use <span> around hint-text to keep the "Hint:" and hint-text on the same line?

Reply:
Thanks! The WYSIWYG produces a `div`, so these elements need to be divs as well. I made them `display: inline` to move them to the same line.